src/jit/SimdArm32Inl.h

```diff
@@ -841,6 +841,11 @@ static bool emitUnaryCondSIMD(sljit_compiler* compiler, Instruction* instr)
     if (instr->info() & Instruction::kIsMergeCompare) {
         Instruction* nextInstr = instr->next()->asInstruction();
 
+        if (*(operands + 1) != VARIABLE_SET_PTR(nullptr)) {
+            args[1].set(operands + 1);
+            sljit_emit_op_flags(compiler, SLJIT_MOV32, args[1].arg, args[1].argw, SLJIT_NOT_EQUAL);
+        }
```

Comment:
I think the code after the if in line 841 could be moved up. There is no duplication that way.